### ISSUE TITLE
fails to build

### ISSUE THREAD

**Issue by japrescott** [felixmosh/bull-board#546]
Hey @felixmosh 

Currently 4.12.2 is failing to build with `bull-board` with only `"@bull-board/express": "~4.12.2"` and `"bullmq": "~3.10.1"` as dependencies
```
node_modules/@bull-board/api/dist/src/queueAdapters/bull.d.ts(1,28): error TS2307: Cannot find module 'bull' or its corresponding type declarations.
```

We only import `import { BullMQAdapter, ExpressAdapter, createBullBoard } from '@bull-board/express';` so `bull` should not be needed.

However, adding `"bull": "~4.10.4"` to the dependencies resolves this issue.

**Comment by felixmosh** [comment]
Hi @japrescott thank you for reporting this issue.

Can you try to change the import to work with 
```
import {ExpressAdapter} from '@bull-board/express';
import {createBullBoard} from '@bull-board/api';
import {BullMQAdapter} from '@bull-board/api/bullMQAdapter';
```

**Comment by mmachatschek** [comment]
@japrescott I'm not sure why exactly you are seeing this issue, we are using this feature in production since the release of the feature 🤔 

our dependencies are `@bull-board/express` and `bullmq`

```ts
  import { ExpressAdapter, createBullBoard, BullMQAdapter } from '@bull-board/express';

  const queues = pseudoCodeReturnSomeQueues();

  const serverAdapter = new ExpressAdapter();

  createBullBoard({
    queues: Object.values(queues).map(queue => new BullMQAdapter(queue())),
    serverAdapter: serverAdapter,
  });
```

Are you 100% sure you are not importing `import { BullAdapter } from '@bull-board/express';` somewhere?

Would be great to have a repo with reproduction steps. I'll happily fix an issue (if one persists).

**Comment by felixmosh** [comment]
It is a typescript issue, that I'm able to reproduce.

**Comment by mmachatschek** [comment]
@felixmosh I'm able to reproduce this to after setting the `skipLibCheck: false` in my tsconfig

**Comment by felixmosh** [comment]
This is the reason I've separated queueAdapters into separate files.

**Comment by mmachatschek** [comment]
I've experimented around a little bit and I see these two options to still support this feature:

1. telling users to use `skipLibCheck: true` in their tsconfig
1. alternative solution:
   1. deprecate the exports of `import { BullMQAdapter, BullMQAdapter } from '@bull-board/express';`
   1. provide exports for both the bullmq adapter and bull:
```ts
import { ExpressAdapter, createBullBoard, BullAdapter } from '@bull-board/express/bull';
import { ExpressAdapter, createBullBoard, BullMQAdapter } from '@bull-board/express/bullMQ';
```
This would be a similar export like it's already done in `@bull-board/api/bullAdapter` and `@bull-board/api/bullMQAdapter`

**Comment by felixmosh** [comment]
I think that we will go with option 2, without re-exporting the adapters (rollback).
If it makes problems more than help, we will require users to declare both of the packages.

**Comment by felixmosh** [comment]
Released in `v5.0.0` since it is a breaking change.

**Comment by japrescott** [comment]
Thank you for a fast resolution and sorry for my radio silence. I should have mentioned I'm on typescript.
